modules/virt-deleting-dvs.adoc

```diff
@@ -5,9 +5,19 @@
 :_content-type: PROCEDURE
 [id="virt-deleting-dvs_{context}"]
 
-= Deleting a data volume
+= Deleting data volumes
 
-You can delete a data volume by using the `oc` command-line interface (CLI).
+You can delete a data volume if needed by using the `oc` command-line interface (CLI), but be aware that doing so also deletes the underlying provisioned volume claim (PVC).
+
+[NOTE]
+====
+When you delete a virtual machine, the data volume it uses is automatically deleted.
+====
+
+[WARNING]
+====
+Removing data volumes manually can produce unexpected results. Instead, use the PVC custom resource (CR) to manage data volumes. In {VirtProductName} release 4.12 and later, automated garbage collection is used to manage data volumes. This transparent process makes manual removal of data volumes unnecessary.
+====
```

Comment:
This is confusing to me.  We should change this warning to read something like the following...

Managing prepared virtual machine disks via a DataVolume (ie. deleting or resizing) will not produce the expected results and should be avoided.  Instead, these operations should target the PVC directly.  In {VirtProductName} release 4.12 and later, automated garbage collection is used to prune completed DataVolumes.

Reply:
I can do this, but let me know if you even want this module to remain at all. If you don't, I will eliminate it (and the module that details listing DVs) entirely for 4.12.